backends-common/rabbitmq/src/main/java/org/apache/james/backend/rabbitmq/RabbitMQManagementAPI.java

```diff
@@ -59,6 +59,9 @@
         @JsonProperty("arguments")
         Map<String, String> arguments;
 
+        @JsonProperty("messages")
+        Long messagesCount;
```

Comment:
Do we need to encode null value ?

Reply:
I think it could be null only with RabbitMQ major version